frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingSkjemaConst.tsx

```diff
@@ -112,7 +114,9 @@ export function utkastDataEllerDefault(
     avtaleId: avtale.id,
     navRegion: defaultNavRegion(avtale, tiltaksgjennomforing),
     navEnheter: defaultNavEnheter(avtale, tiltaksgjennomforing),
-    administratorer: tiltaksgjennomforing?.administratorer?.map((admin) => admin.navIdent),
+    administratorer:
+      tiltaksgjennomforing?.administratorer?.map((admin) => admin.navIdent) || [ansatt.navIdent] ||
+      [],
```

Comment:
Denne siste `|| []` vil vel aldri slå inn?

Reply:
Godt poeng. Jeg la bare litt ukritisk inn samme linje som for avtale, men burde nok heller slettet den fra avtale. Jeg fikser.